tidb-configuration-file.md

```diff
@@ -284,6 +284,12 @@ Configuration items related to performance.
 - Default value: `0`
 - This configuration only takes effect when `prepared-plan-cache.enabled` is `true`. When the size of the LRU is greater than `prepared-plan-cache.capacity`, the elements in the LRU are also removed.
 
+### `txn-total-size-limit`
+
+- The size limit of a single transaction in TiDB.
+- Default value: `104857600` (in bytes)
+- In a single transaction, the total size of key-value records cannot exceed this value. The upper limit of this configuration value is `10737418240` (10 GB). Note that if you have used the binlog whose downstream consumer is Kafka (such as the `arbiter` cluster), the value of this parameter must be no more than `1073741824` (1 GB). This is because 1 GB is the upper limit of a single message size that Kafka can process. In this case, an error is returned if this limit is exceeded.
```

Comment:
```suggestion
- In a single transaction, the total size of key-value records cannot exceed this value. The maximum value of this parameter is `10737418240` (10 GB). Note that if you have used the binlog to serve the downstream consumer Kafka (such as the `arbiter` cluster), the value of this parameter must be no more than `1073741824` (1 GB). This is because 1 GB is the upper limit of a single message size that Kafka can process. Otherwise, an error is returned if this limit is exceeded.
```